packages/server/package.json

```diff
@@ -37,16 +38,14 @@
     "@josephg/resolvable": "^1.0.0",
     "@types/express": "4.17.13",
     "@types/express-serve-static-core": "4.17.28",
-    "@types/lru-cache": "^7.6.1",
```

Comment:
@trevor-scheer Was this intentional?

Reply:
or well, I think it makes sense to drop it from packages/server but it needs to pop up in top-level package.json?

Reply:
Oh I see, it actually should be built in now. Wonder why I'm getting issues.